config/collections.yml

```diff
@@ -2,448 +2,529 @@ default:
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.kubernetes
-    Regex: kubernetes|k8s
-    NewCollection: 'FALSE'
-    MergeKey: kubernetes
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: kubernetes.core
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: kubernetes
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.kubevirt
-    Regex: kubevirt
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.libvirt
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.molecule
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.mongodb
-    Regex: mongo
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.mysql
-    Regex: mysql
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.network
-    Regex: /network/
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.okd
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.postgresql
-    Regex: postgres
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.proxysql
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.rabbitmq
-    Regex: rabbitmq
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.routeros
-    Regex: routeros
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: skydive
-    Regex: skydive
-    NewCollection: 'FALSE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.sops
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: community.vmware
-    Regex: vmware
-    NewCollection: 'FALSE'
-    MergeKey: vmware
+    in_package: yes
   - Site: https://github.com
     Org: ansible-collections
     Repo: vmware_rest
-    Regex: vmware
-    NewCollection: 'FALSE'
-    MergeKey: vmware
+    in_package: yes
```

Comment:
`vmware_rest` is not currently in `ansible` package, though should be for ansible 5.0.0

Reply:
Thanks I'll set it to no for now then, we can change it later